websites/M/Mastodon/presence.ts

```diff
@@ -84,26 +93,37 @@ presence.on("UpdateData", async () => {
 			presenceData.details = "Reading Privacy and Policy";
 			break;
 		}
+		case "/follow_requests": {
+			presenceData.smallImageText = "follow_requests";
```

Comment:
Considering that there is no `smallImageKey` set for this path (and many others), remove all unused `smallImageText` lines - they are effectively redundant comments.